From apgo@patchbomb.org Wed Jan 18 18:14:50 2006
Date: Wed, 18 Jan 2006 21:12:57 -0500
From: apgo@patchbomb.org (Arthur Othieno)
To: gregkh@suse.de
Cc: <ricknu-0@student.ltu.se>, <apgo@patchbomb.org>
Subject: PCI: cyblafb: remove pci_module_init() return, really.
Message-ID: <20060119021257.GB23678@krypton>
Content-Disposition: inline

Richard Knutsson <ricknu-0@student.ltu.se> did the original pci_module_init()
cleanups:

    http://marc.theaimsgroup.com/?l=linux-kernel&m=113330872125068&w=2
    http://marc.theaimsgroup.com/?l=linux-kernel&m=113330888507321&w=2

Greg, on it's way upstream, pci_module_init() return sneaked back in for
cyblafb?

    http://marc.theaimsgroup.com/?l=linux-pci&m=113652969209562&w=2
    http://marc.theaimsgroup.com/?l=linux-pci&m=113683930220421&w=2

Remove for good.

Signed-off-by: Arthur Othieno <apgo@patchbomb.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/video/cyblafb.c |    1 -
 1 file changed, 1 deletion(-)

--- gregkh-2.6.orig/drivers/video/cyblafb.c
+++ gregkh-2.6/drivers/video/cyblafb.c
@@ -1665,7 +1665,6 @@ static int __devinit cyblafb_init(void)
 		}
 #endif
 	output("CyblaFB version %s initializing\n", VERSION);
-	return pci_module_init(&cyblafb_pci_driver);
 	return pci_register_driver(&cyblafb_pci_driver);
 }
 
